doc/source/development.rst

```diff
@@ -28,162 +32,166 @@ Fork the upstream KIWI repository
 Create a local clone of the forked KIWI repository
 --------------------------------------------------
 
-.. code:: bash
+.. code-block:: shell-session
 
     $ git clone https://github.com/YOUR-USERNAME/kiwi
 
     $ git remote add upstream https://github.com/SUSE/kiwi.git
 
-Create a python virtual development environment
------------------------------------------------
-
-The Python project uses :command:`pyvenv` to setup a development environment
-for the desired Python version. The script :command:`pyvenv` is already
-installed when using Python 3.3 and higher (see
-https://docs.python.org/3.3/whatsnew/3.3.html#pep-405-virtual-environments
-for details). For Python 2.7 use :command:`virtualenv`, which is provided
-via pip or as an extra package in your favourite Linux distribution.
+Install required operating system packages
+------------------------------------------
 
-However, for setting up a Python virtual development environment the
-following additional LaTeX, include, header files and compilers are required
-in order to allow for compiling the C parts of the runtime required
-Python modules:
+KIWI requires the following additional packages which are not provided by
+:command:`pip`:
 
-* XML processing with libxml2 and libxslt (for :mod:`lxml`)
-* Foreign function interface library (libffi48)
-* Python header files (for :mod:`xattr`)
-* GCC compiler and glibc-devel header files
-* LaTeX packages for building PDF documentation
+* XML processing libraries: libxml2 and libxslt (for :mod:`lxml`)
+* Python header files (for :mod:`xattr`), usually provided by a package
+  called `python-devel` or `python3-devel`
+* The `enchant` spell checking library
+* gcc compiler and glibc-devel header files
+* The `ShellCheck <https://github.com/koalaman/shellcheck>`_ script
+  linter.
+* A manipulation program for ISO images, either: ``xorriso`` (preferred) or
+  ``genisoimage``.
 
-.. note::
+A full LaTeX installation is required to build the PDF documentation
+[#f1]_.
 
-    On SUSE systems the required components can be installed
-    with the following command. Package names and install command
-    are different on other systems.
+On SUSE based systems run:
 
-.. code:: bash
+.. code-block:: shell-session
 
     $ zypper install \
```

Comment:
Good idea! I'll try that if it installs everything with `--no-recommends`.

Reply:
That list was suspiciously short and missing **a lot** of packages.

Reply:
Usually when I use this option, a lot of texlive packages are missing -- which was to be expected. :wink:  Not sure if it is useful to add the missing packages into the above line.

I thought it would be a nice idea, but if that doesn't work, just leave it as is. :-)

Reply:
I'd say let's keep it in since we already have a similar list for Fedora. It might be useful in case someone wants to build the pdf version of the documentation.